src/main/java/step3/GameBoard.java

```diff
@@ -27,11 +29,16 @@ public Count registerTrialCount() {
         return INPUT_VIEW.inputValue();
     }
 
-    public void renderRaceProgress(RacingCarGroup carGroup) {
+    public void renderRaceResult(List<Record> recordList) {
+        recordList.forEach(r -> {
+            renderPosition(r);
+        });
+    }
 
-        carGroup.carsCurrentPosition().stream()
-                .forEach(pos -> {
-                    render(appendText(pos.getCount()));
+    private void renderPosition(Record record) {
+        record.recordList().stream()
+                .forEach(p -> {
+                    System.out.println(appendText(p.getPosition()));
```

Comment:
System.out.println을 최소화하려고 StringBuilder를 사용했는데 이래서는 자동차가 n^2대일 경우 System.out도 n^2호출 될 것 같네요. StringBuilder를 최대한 활용하면 좋지 않을까요? 
```suggestion
    private void renderPosition(Record record){
        STRING_BUILDER.setLength(ZERO);

        for (Position position : record.recordList()) {
            appendSkidMarkV1(position);
        }
        
        System.out.println(STRING_BUILDER);
    }

    private void appendSkidMarkV1(Position position) {
        for(int i = 0; i < position.getPosition(); ++i) {
            STRING_BUILDER.append(UNIT_OF_MOVE);
        }
        STRING_BUILDER.append(System.lineSeparator());
    }
    private void appendSkidMarkV2(Position position) {
        STRING_BUILDER.append(String.join("", Collections.nCopies(position.getPosition(), UNIT_OF_MOVE)))
                .append(System.lineSeparator());
    }
    
    
```

이런 방식들도 있겠죠 ㅎ
간접적인 제안이아니라 이렇게 직접 코드를 작성해서 몇가지의 대안책들을 보여드린 이유는, 
이처럼 최대한 반복을 줄이고 재활용을 하는 메서드를 작성하기를 바라며 하나의 관점을 보여드린 것입니다. 
이것보다 더 좋은 코드들도 많을꺼에요.